RELEASE_NOTES

```diff
@@ -9,6 +9,14 @@ commit history located at https://github.com/apache/madlib/commits/master.
 
 Current list of bugs and issues can be found at https://issues.apache.org/jira/browse/MADLIB.
 
+—-------------------------------------------------------------------------
+MADlib v1.17:
+
+Release Date:
+
+Other:
+    - DL: Supported keras version is fixed to 2.2.4
```

Comment:
1. Do we only support one keras version? I think we support 2.2.4 and lower but not sure what's the lowest version we support
1. Same goes for tensorflow, we support 1.14 and lower but not sure about the lowest version we support